src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,16 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+// @codingStandardsIgnoreFile
+namespace Cake\Network\Email;
+
+class_alias(\Cake\Mailer\AbstractTransport::class, AbstractTransport::class);
+
+if (class_exists(AbstractTransport::class)) {
+    return;
+}
```

Comment:
> merge it and let's find out

That's not the right approach, we have a responsibility to try and ensure we don't break people's apps.